src/components/Auth0Context.svelte

```diff
@@ -32,7 +33,7 @@
     let tokenRefreshIntervalId;
 
     // getContext doesn't seem to return a value in OnMount, so we'll pass the auth0Promise around by reference.
-    let auth0Promise = createAuth0Client({domain, client_id});
+    let auth0Promise = createAuth0Client({domain, client_id, audience});
```

Comment:
Does this still play nice if the audience isn't set?

Reply:
Yes. I noticed that Auth0 defined `audience?: string;` so being an optional value, it does give tokens in either case, but with audience that token is a JWT.